src/components/DroitsDetails.vue

```diff
@@ -126,23 +126,38 @@
             class="aj-droit-content-buttons-cta"
             v-bind:benefit="droit"
           ></BenefitCta>
-        </div>
 
-        <a
-          v-if="droit.msa"
-          target="_blank"
-          rel="noopener"
-          class="aj-droit-pro-agricole"
-          href="http://www.msa.fr/lfr/web/msa/espace-prive"
-          v-analytics="{
-            name: droit.label,
-            action: 'msa',
-            category: 'General',
-          }"
-        >
-          <img src="@/assets/images/doigt.svg" /> Démarches pour les professions
-          agricoles
-        </a>
+          <a
+            v-if="droit.msa"
+            target="_blank"
+            rel="noopener"
+            class="aj-droit-pro-agricole"
+            href="http://www.msa.fr/lfr/web/msa/espace-prive"
+            v-analytics="{
+              name: droit.label,
+              action: 'msa',
+              category: 'General',
+            }"
+          >
+            <img src="@/assets/images/doigt.svg" /> Démarches pour les
+            professions agricoles
+          </a>
+
+          <div class="is-align-vertically-center">
+            <a
+              class="text-center"
+              @click="alertBrokenLink()"
+              v-if="brokenLinkButtonState === 'show'"
+              >Lien invalide ? Cliquer ici pour nous notifier !</a
+            >
+            <span
+              class="text-center"
```

Comment:
idem, étant donné que c'est dans une nouvelle classe `is-align-vertically-center`. :thinking: